task16/css/style.css

```
@@ -0,0 +1,146 @@
+body {
+    height: 100vh;
+    padding: 60px;
+    background-color: rgb(234, 235, 236);
+    font-family:Arial, Helvetica, sans-serif;
+    letter-spacing: .1em;
+}
+
+.loader,
+.loader:before,
+.loader:after {
+  background: #ffffff;
+  -webkit-animation: load1 1s infinite ease-in-out;
+  animation: load1 1s infinite ease-in-out;
+  width: 1em;
+  height: 4em;
+}
+.loader {
+  color: #ffffff;
+  text-indent: -9999em;
+  margin: 88px auto;
+  position: relative;
+  font-size: 11px;
+  -webkit-transform: translateZ(0);
+  -ms-transform: translateZ(0);
+  transform: translateZ(0);
+  -webkit-animation-delay: -0.16s;
+  animation-delay: -0.16s;
+}
+.loader:before,
+.loader:after {
+  position: absolute;
+  top: 0;
+  content: '';
+}
+.loader:before {
+  left: -1.5em;
+  -webkit-animation-delay: -0.32s;
+  animation-delay: -0.32s;
+}
+.loader:after {
+  left: 1.5em;
+}
+@-webkit-keyframes load1 {
+  0%,
+  80%,
+  100% {
+    box-shadow: 0 0;
+    height: 4em;
+  }
+  40% {
+    box-shadow: 0 -2em;
+    height: 5em;
+  }
+}
+@keyframes load1 {
+  0%,
+  80%,
+  100% {
+    box-shadow: 0 0;
+    height: 4em;
+  }
+  40% {
+    box-shadow: 0 -2em;
+    height: 5em;
+  }
+}
+
+.tab-menu {
+   max-width: 760px;
+   margin: 0 auto;
+   overflow: hidden; 
+   display: flex;
+   justify-content: space-between;
+}
+
+.menu-list {
+    padding: 15px 0 15px;
+    width: 188px;
+    text-align: center;
+    color: #fff;
+    background-color: rgb(8, 2, 58);
+    border-radius: 10px 10px 0 0;
+    font-weight: bold;
+    font-size: 15px;
+    text-decoration: none;
+}
+.menu-list.active {
+    background: #fff;
+    color: rgb(8, 2, 58);
+}
+
+.menu-list:hover {
+    text-decoration: underline;
+}
+
+.tab-container {
+    max-width: 760px;
+    min-height: 400px;
+    margin: 0 auto;
+    background: #fff;
+    box-sizing: border-box;
+    padding: 20px 35px;
+    display: block;
+}
+
+.news-wrapper.show {
+    display: flex;
+}
+
+.news-wrapper {
+    display: none;
+}
+
+.news-img {
+    margin: 20px;
+    width: 280px;
+}
+
+.news-title-item {
+    margin: 30px 0 15px 40px;
+    list-style: square;
+}
```

NITS
In this case, the `margin-bottom` is only 30px due to the margin offset.
It also creates an extra margin(40px) under the fourth `li`.

In addition, `margin-top` and `margin-left` should be specified for the `ul`, not the `li`, for the following reasons.
- Because setting the style on `ul` makes `li` and `a` more reusable.
- Because it is considered a positioning setting for the `ul` element, not for the `li` alone.

```suggestion
.news-title-list {
    margin-top: 30px;
    margin-left: 40px;
}
.news-title-item {
    list-style: square;
}
.news-title-item:nth-of-type(n+2) {
    margin-top: 45px;
}
```

---

Eliminate unnecessary styles
And I learned a lot about the reusability of CSS!
Thank you for your suggestions.
[f740b9d]